project3/styles.css

```diff
@@ -0,0 +1,424 @@
+    font-size: inherit;
+
+}
+
+legend {
+    margin-bottom: 0;
+}
+
+
+label {
+    display: block;
+
+}
+
+label[for=strength],
+label[for=speed],
+label[for=intelligience], {
+    text-align: center;
+    margin-bottom: 30px;
+}
+
+select {
+    cursor: pointer;
+}
+
+textarea {
+    width: 100%;
+}
+
+input {
+    margin-bottom: 15px;
+    width: 100%;
+
+}
+
+
+select {
+    background: gray;
+    border: 0;
+    cursor: pointer;
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: inherit;
+    padding: 20px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 40px;
+    text-align: center;
+    width: 100%;
+
+}
+
+
+input[type="radio"],
+input[type="checkbox"] {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    border: 2px solid var(--color-lightgray);
+    cursor: pointer;
+    height: 20px;
+    width: 20px;
+}
+
+input[type="radio"]:checked,
+input[type="checkbox"]:checked {
+    background-color: aqua;
+    display: inline-block;
+}
+
+
+input[type="radio"] {
+    border-radius: 50%
+}
+
+input[type="radio"]:hover,
+input[type="checkbox"]:hover {
+    border: 2px solid var(--color-lime);
+    opacity: 1;
+}
+
+
+input[type="radio"]:checked+label,
+input[type="checkbox"]:checked+label {
+    color: var(--color-lime);
+}
+
+
+
+input[type="text"]:focus,
+input[type="email"]:focus,
+input[type="number"]:focus {
+    background-color: aqua;
+}
+
+input[type="text"]:required:valid,
+input[type="email"]:required:valid,
+input[type="number"]:required:valid {
+    background-color: greenyellow;
+
+
+}
+
+input[type="text"]:not(:placeholder-shown):invalid,
+input[type="email"]:not(:placeholder-shown):invalid,
+input[type="number"]:not(:placeholder-shown):invalid {
+    background-color: orange;
+}
+
+
+#age {
+    max-width: 80px;
+}
+
+.category label:hover,
+.gender label:hover {
+    border-bottom: 3px solid var(--color-lime);
+    display: inline-block;
+}
+
+
+
+
+
+
+.category input,
+.category label,
+.gender input,
+.gender label {
+    vertical-align: top;
+}
+
+
+
+
+/* .category,
+.gender {
+    width: 10em;
+    overflow: auto;
+}
+
+
+.category input,
+.gender input {
+    margin: 8px;
+}
+
+
+.category label,
+.gender label {
+    width: 12em;
+    float: right;
+} */
+
+
+
+
+.confirmation {
+    border-top: 5px solid var(--color-blue);
+    display: block;
+    overflow: auto;
+    padding: 40px;
+}
+
+.confirmation input {
+    margin: 20px;
+    vertical-align: middle;
+
+}
+
+.confirmation label {
+    text-align: center;
+}
+
+
+
+.register-now {
+    border: 5px transparent;
+}
+
+.register-now:hover {
+    border: 10px solid var(--color-lime);
+}
+
+
+
+
+.button-reset {
+    background: transparent;
+    border: 0;
+    cursor: pointer;
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: inherit;
+    padding: 0;
+    text-align: inherit;
+}
+
+.button-reset[disabled] {
+    cursor: default;
+}
+
+
+.button-reset,
```

Comment:
Here you can leave off the `.button-reset,` class and just style the specific button classname `.submit`. This means you can use the `button-reset `class to reset all sorts of buttons, not just this one, which is useful in a large project.